tests/text2sql/test_text2sql.sh

```diff
@@ -13,7 +13,7 @@ export TAG='comps'
 
 export TEXT2SQL_PORT=11700
 export LLM_ENDPOINT_PORT=11710
-
+export DATA_PATH=${model_cache}
```

Comment:
remove line 10, 
export DATA_PATH=${model_cache:-$WORKPATH/data}  
BTY, this env are not correctly used in the test, need to fix.